lib/ansible/galaxy/collection/__init__.py

```diff
@@ -409,6 +409,7 @@ def install_collections(
         force_deps,  # type: bool
         allow_pre_release,  # type: bool
         artifacts_manager,  # type: ConcreteArtifactsManager
+        upgrade=False,  # type: bool
```

Comment:
For consistency, I think it's better not to assign defaults:
```suggestion
        upgrade,  # type: bool
```
Also, could you move it after `force_deps`? These flags feel related so it'd be nice to group them.

Reply:
Will do. I assigned defaults so I didn't need to edit unit tests yet again. :laughing: